samples/hello/src/com/microsoft/aad/adal/hello/MainActivity.java

```diff
@@ -302,15 +260,17 @@ private void displayMessage(String msg) {
 
     private String getUserId() {
         if (mResult != null && mResult.getUserInfo() != null
-                && mResult.getUserInfo().getUserId() != null) {
-            return mResult.getUserInfo().getUserId();
+                && mResult.getUserInfo().getUniqueId() != null) {
+            return mResult.getUserInfo().getUniqueId();
```

Comment:
On ADAL.NET the unique ID is some ugly non-human readable string (the subject or object id). Does that map to the previous functionality of getUserId() here? I would think displayable id would be closer to what you want.